test/example/stack-utils.txt

```diff
@@ -0,0 +1,16 @@
+;; Test compact
+(block
+ (nop)
+ (i32.const 0)
+ (nop)
+ (i64.const 0)
+ (nop)
+ (nop)
+)
+(block
+ (i32.const 0)
+ (i64.const 0)
+)
+;; Test stack signature composition
```

Comment:
Why aren't these printed?

Reply:
These tests are just a bunch of asserts, so there is nothing to be printed. In contrast, the test that removed the nops was easier to write as a couple of prints rather than as a bunch of asserts.